src/nodes.coffee

```diff
@@ -1482,6 +1482,9 @@ exports.Param = class Param extends Base
   constructor: (@name, @value, @splat) ->
     if (name = @name.unwrapAll().value) in STRICT_PROSCRIBED
       @name.error "parameter name \"#{name}\" is not allowed"
+    if @name instanceof Obj and @name.generated
+      token = @name.objects[0].operatorToken
```

Comment:
is there no way that part of this could be null? (I suppose we don't have any degenerate case where we insert an empty object?)